docs/operations.rst

```diff
@@ -443,3 +443,108 @@ The reported value will be the weighted mean value of the last 30s.
       min: 10
       min %: 5
     - refresh action: 01:00   # ... but refresh every hour
+
+
+Home Assistant MQTT Sensors
+--------------------------------------
+
+This will add MQTT Sensors to `Home Assistant <https://www.home-assistant.io/>`_. One can add this to a `package <https://www.home-assistant.io/docs/configuration/packages/>`_.
+
+.. code-block:: yaml
+
+   mqtt:
+     sensor:
+       - name: "sml01_energy_total"
+         unique_id: f855b69a-828b-4cad-bafd-7563a909153e
+         state_topic: "sml2mqtt/sml01/energy_total"
+         icon: mdi:transmission-tower
+         unit_of_measurement: kWh
+         state_class: total
+         device_class: energy
+         availability:
+           topic: "sml2mqtt/status"
+           payload_available: "ONLINE"
+           payload_not_available: "OFFLINE"
+       - name: "sml01_energy_total_returned"
+         unique_id: 855b9e06-9d39-41e9-afc0-ee0f567aacb2
+         state_topic: "sml2mqtt/sml01/energy_total_returned"
+         icon: mdi:transmission-tower
+         unit_of_measurement: kWh
+         state_class: total
+         device_class: energy
+         availability:
+           topic: "sml2mqtt/status"
+           payload_available: "ONLINE"
+           payload_not_available: "OFFLINE"
+         operations:
+           refresh action: 24:00
```

Comment:
What does this do?

Reply:
Uh, good catch. That's an sml2mqtt config, not home-assistant. Should be removed.